infra/benchmark-run-ddata.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+declare -r CLAZZ=com.lightbend.akka.bench.ddata.DistributedDataBenchmark
+
+fix --verbose --concurrency 2 \
+  nodes_with_role:benchmark \
+  ssh:"cd /home/akka/apps; sudo -u akka /home/akka/sbt ';project apps; runMain $CLAZZ'"
```

Comment:
Check this out @patriknw @2m et al, this is how we can run an app on all nodes in parallel. 
The concurrency setting must be equal (well or greater than) the number of nodes we want to run this on.

Results are printed to STDOUT collected from the nodes we have executed this on, so e.g.:

```
[akka-node-002] out: [INFO] [06/14/2017 04:10:55.613] [Thread-11] [StarvationDetector$StarvationDetectorThread(akka://BenchmarkSystem)] Starvation detector starting for dispatcher [Dispatcher[akka.actor.default-dispatcher]]
[akka-node-001] out: [INFO] [06/14/2017 04:10:56.017] [BenchmarkSystem-akka.actor.default-dispatcher-4] [akka://BenchmarkSystem@10.132.0.7:2551/user/coordinatorManager/singleton] DData disseminated to 2 nodes in 498 ms.
[akka-node-001] out: [INFO] [06/14/2017 04:10:56.517] [BenchmarkSystem-akka.actor.default-dispatcher-9] [akka://BenchmarkSystem@10.132.0.7:2551/user/coordinatorManager/singleton] DData disseminated to 2 nodes in 499 ms.
[akka-node-001] out: [INFO] [06/14/2017 04:10:57.017] [BenchmarkSystem-akka.actor.default-dispatcher-2] [akka://BenchmarkSystem@10.132.0.7:2551/user/coordinatorManager/singleton] DData disseminated to 2 nodes in 500 ms.

...

[akka-node-001] out: === Distributed Data Benchmark 50%ile: 499967 µs, 90%ile: 523007 µs, 99%ile: 996863 µs
[akka-node-001] out: Histogram of dissemination latencies in microseconds.
[akka-node-001] out:        Value     Percentile TotalCount 1/(1-Percentile)
[akka-node-001] out:
[akka-node-001] out:   485375.000 0.000000000000          1           1.00
[akka-node-001] out:   485375.000 0.100000000000          1           1.11
[akka-node-001] out:   491007.000 0.200000000000          2           1.25
[akka-node-001] out:   491519.000 0.300000000000          3           1.43
[akka-node-001] out:   498687.000 0.400000000000          4           1.67
[akka-node-001] out:   499967.000 0.500000000000          5           2.00
[akka-node-001] out:   500479.000 0.550000000000          8           2.22
[akka-node-001] out:   500479.000 0.600000000000          8           2.50
[akka-node-001] out:   500479.000 0.650000000000          8           2.86
[akka-node-001] out:   500479.000 0.700000000000          8           3.33
[akka-node-001] out:   500479.000 0.750000000000          8           4.00
[akka-node-001] out:   500479.000 0.775000000000          8           4.44
[akka-node-001] out:   500479.000 0.800000000000          8           5.00
[akka-node-001] out:   523007.000 0.825000000000          9           5.71
[akka-node-001] out:   523007.000 0.850000000000          9           6.67
[akka-node-001] out:   523007.000 0.875000000000          9           8.00
[akka-node-001] out:   523007.000 0.887500000000          9           8.89
[akka-node-001] out:   523007.000 0.900000000000          9          10.00
[akka-node-001] out:   996863.000 0.912500000000         10          11.43
[akka-node-001] out:   996863.000 1.000000000000         10
[akka-node-001] out: #[Mean    =   548646.400, StdDeviation   =   149618.061]
[akka-node-001] out: #[Max     =   996863.000, Total count    =           10]
[akka-node-001] out: #[Buckets =           15, SubBuckets     =         2048]
```